hw_7_classes.py

```diff
@@ -0,0 +1,233 @@
+import json
+import pprint
+import os
+import pathlib
+import fnmatch
+
+
+# Задача-1
+# У вас есть список(list) IP адресов. Вам необходимо создать
+# класс который будет иметь методы:
+# 1) Получить список IP адресов
+# 2) Получить список IP адресов в развернутом виде
+# (10.11.12.13 -> 13.12.11.10)
+# 3) Получить список IP адресов без первых октетов
+# (10.11.12.13 -> 11.12.13)
+# 4) Получить список последних октетов IP адресов
+# (10.11.12.13 -> 13)
+
+
+class IpAddress:
+
+    ip_list = ['10.11.12.13', '192.168.12.1', '147.22.243.33']
+
+    def __init__(self):
+        self._list = IpAddress.ip_list
+        self._reversed_addr = []
+        self._last_3_ocktets = []
+        self._last_ocktets = []
+
+    def get_ip_list(self):
+        return f"IP addresses: {self._list}"
```

Comment:
better to return some value, not msg. If can't use this method in your code `list_of_ips = self. get_ip_list()` will store str msg

Reply:
Same for all methods in class